components/user/UserMilestone.js

```diff
@@ -1,22 +1,50 @@
 import Icon from "../Icon";
+import { VerticalTimelineElement } from 'react-vertical-timeline-component'
+import 'react-vertical-timeline-component/style.min.css'
+import Image from "next/image";
 
 export default function UserMilestone({ milestone }) {
+  function getStyle() {
+    if (milestone.icon === 'github') return 'black'
+    else return milestone.color
```

Comment:
This doesn't follow the projects coding style

I think either use a ternary or write the if/else out in full

Reply:
As it is only used once in the code, it could be done inline with a ternary